scripts/variantstore/wdl/GvsCreateTables.wdl

```diff
@@ -2,47 +2,50 @@ version 1.0
 
 workflow CreateBQTables {
 
-    input {
-        Int max_table_id
-        String project_id
-        String dataset_name
-        String? service_account_json_path
-        String pet_schema_json = '[{"name": "location","type": "INTEGER","mode": "REQUIRED"},{"name": "sample_id","type": "INTEGER","mode": "REQUIRED"},{"name": "state","type": "STRING","mode": "REQUIRED"}]'
-        String vet_schema_json = '[{"name": "sample_id", "type" :"INTEGER", "mode": "REQUIRED"},{"name": "location", "type" :"INTEGER", "mode": "REQUIRED"},{"name": "ref", "type" :"STRING", "mode": "REQUIRED"},{"name": "alt", "type" :"STRING", "mode": "REQUIRED"},{"name": "AS_RAW_MQ", "type" :"STRING", "mode": "NULLABLE"},{"name": "AS_RAW_MQRankSum", "type" :"STRING", "mode": "NULLABLE"},{"name": "QUALapprox", "type" :"STRING", "mode": "NULLABLE"},{"name": "AS_QUALapprox", "type" :"STRING", "mode": "NULLABLE"},{"name": "AS_RAW_ReadPosRankSum", "type" :"STRING", "mode": "NULLABLE"},{"name": "AS_SB_TABLE", "type" :"STRING", "mode": "NULLABLE"},{"name": "AS_VarDP", "type" :"STRING", "mode": "NULLABLE"},{"name": "call_GT", "type" :"STRING", "mode": "NULLABLE"},{"name": "call_AD", "type" :"STRING", "mode": "NULLABLE"},{"name": "call_GQ", "type" :"INTEGER", "mode": "NULLABLE"},{"name": "call_PGT", "type" :"STRING", "mode": "NULLABLE"},{"name": "call_PID", "type" :"STRING", "mode": "NULLABLE"},{"name": "call_PL", "type" :"STRING", "mode": "NULLABLE"}]'
-        String ref_ranges_schema_json = '[{"name": "location","type": "INTEGER","mode": "REQUIRED"},{"name": "sample_id","type": "INTEGER","mode": "REQUIRED"},{"name": "length","type": "INTEGER","mode": "REQUIRED"},{"name": "state","type": "STRING","mode": "REQUIRED"}]'
-        Int? preemptible_tries
-    }
-
-    call CreateTables as CreateVetTables {
-        input:
-        project_id = project_id,
-        dataset_name = dataset_name,
-        datatype = "vet",
-        max_table_id = max_table_id,
-        schema_json = vet_schema_json,
-        superpartitioned = "true",
-        partitioned = "true",
-        service_account_json_path = service_account_json_path,
-        preemptible_tries = preemptible_tries
-    }
-
-    call CreateTables as CreateRefRangesTables {
-        input:
-            project_id = project_id,
-            dataset_name = dataset_name,
-            datatype = "ref_ranges",
-            max_table_id = max_table_id,
-            schema_json = ref_ranges_schema_json,
-            superpartitioned = "true",
-            partitioned = "true",
-            service_account_json_path = service_account_json_path,
-            preemptible_tries = preemptible_tries
-    }
-
-    output {
-      String vetDone = CreateVetTables.done
-      String refDone = CreateRefRangesTables.done
-    }
+  input {
+    String dataset_name
+    String project_id
+
+    Int max_table_id
+
+    Int? preemptible_tries
```

Comment:
Same ToL as above… maybe we shouldn't even do this with PE VMs… the benefit of PE is cost, and the downside is that we could get pre-empted in the middle of doing something and retry